### DISCUSSION TITLE
preguntas sobre el proyecto

### DISCUSSION THREAD

**Discussion by Juliaarias22** [PatronesHermosos/design-thinking-1ra-generacion#5] [Q&A]
El numero de la sala que estábamos, es el numero de equipo que tenemos? @KasturiSShah

**Comment by KasturiSShah** [comment]
Hola @Juliaarias22 
Claro, tú y Carolina estuvieron en la sala 8, entonces, forman el equipo 8.
¡Gracias!

**Reply by Juliaarias22** [reply]
Gracias Kasturi!!

**Comment by Juliaarias22** [comment]
¿Donde encuentro las tareas y módulos que tenemos que hacer?

**Reply by KasturiSShah** [reply]
Hola @Juliaarias22 
Toda la información está disponible en la plataforma. Instrucciones para acceder a la plataforma están en nuestros correos semanales. Algunos tips:
Inicia sesión aquI: https://cursos.patroneshermosos.com/
Completa cada lección dentro de cada módulo.
Espero te ayude,
Kasturi

**Reply by Gaga1982** [reply]
@KasturiSShah Buenas noches. Creo que estaba en el equipo 6 (trabajo sola), por lo menos eso mencionaste en la asesoría del jueves 2 de Marzo. Pero como no tiene nombre la carpeta me quede con esa duda. o.O

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 

Así es, tu número de equipo es 6. Entonces, tienes que rellenar el formulario dentro de la carpeta que corresponde a tu numero de equipo, es decir, este formulario: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/tree/main/equipo-06/etapa-0

Espero te ayude
Kasturi

**Reply by Gaga1982** [reply]
Gracias Kastuti. Lo modifique a las 19 hs y no me sale. No se xq si le di guardar.

**Reply by MaryAnnHernani** [reply]
Buen día, disculpen no pude acceder a la sesión del fin de semana no tengo grupo aún quizás pueda agregarme a alguno. Muchas gracias.

**Reply by KasturiSShah** [reply]
Hola @MaryAnnHernani 
No hay ningún problema. Las diapositivas para la sesión están en la plataforma. ¿Quieres completar el proyecto sola o en equipo? Hay varios factores en esta decisión; una de ellas que discutí con varias personas en las sesiones interactivas es la disponibilidad para coordinar con miembros del equipo que viven en diferentes zonas horarias para poder completar las tareas. Si prefieres discutir por correo, mándame correo preguntas@patronas.org
Kasturi

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 
¿Podrías darme más detalle? Picaste el lapicero en la liga que te mandaste, intentaste hacer cambios y picaste el botón verde, o algo así? Si tienes capturas de pantallas de tu proceso de entrega que me podrías enviar, sería de gran ayuda.
Kasturi

**Reply by Gaga1982** [reply]
Si vos sabes que me re costo porque me daba error cuando entraba en la carpeta. Y me salía el lápiz en gris. Hasta que puse modificar y lo pude cargar, complete todo y le di el botón verde o sea lo guarde  desde la PC y yo pensé que había quedado guardado por eso no hice capturas de pantalla. Lo hice el domingo 5 como a las 19 hs CDMX. Y después contestaste me fije por el celular acá en GitHub que tengo la app descargada y no me sale 🤷🏻‍♀️

**Reply by KasturiSShah** [reply]
He checado los permisos de tu cuenta y creo que todo está en orden: tu cuenta tiene acceso a este repositorio. Tengo dos sugerencias:
1. ¿Podrías enviarme tu entrega por este medio? Es decir, ¿podrías escribir las respuestas a los puntos del formulario aquí? Subiré esa información a tu carpeta en Github para que puedas adelantar con la etapa 1 del proyecto.
2. ¿Podrías asistir a una asesoría esta semana? Dedicaremos unos minutos para resolver dudas, quizás sería más fácil resolverlo en vivo cuando estés compartiendo pantalla. 

Gracias!!
Kasturi

**Reply by Gaga1982** [reply]
Ok. Ahora te quiero.mandar el.error y me.sale.esto ![Screenshot_20230306_192818_GitHub.jpg](https://user-images.githubusercontent.com/100386928/223260492-762382fa-e80f-48b7-8863-956a7cbc207c.jpg)

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 
Creo que archivos .heic tienen vídeos cortos. ¿Podrías convertirlo a un archivo de imagen, ej., JPG o PNG? Y luego subirlo? Github no acepta vídeos.
Kasturi

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 
Un recordatorio por favor: mándame tu etapa 0 para que pueda darte retroalimentación tan pronto como sea posible. Si me mandas todo por correo, subiré tu entrega a Github en tu carpeta individual.
Kasturi

**Comment by Gaga1982** [comment]
Si. Perdón ya te lo mando recién llego de trabajar El 6 mar 2023 21:02, KasturiSShah ***@***.***> escribió:
Hola @Gaga1982
Creo que archivos .heic tienen vídeos cortos. ¿Podrías convertirlo a un archivo de imagen, ej., JPG o PNG? Y luego subirlo? Github no acepta vídeos.
Kasturi

**Reply by KasturiSShah** [reply]
Gracias, cuando puedas.
